src/Microsoft.VisualStudio.AppDesigner/PropPageDesigner/PropPageDesignerWindowPane.vb

```diff
@@ -51,7 +51,7 @@ Namespace Microsoft.VisualStudio.Editors.PropPageDesigner
 
             Dim DesignerView As PropPageDesignerView = GetPropPageDesignerView()
             If DesignerView IsNot Nothing Then
-                Dim KeyCode As Keys = DirectCast(m.WParam.ToInt32(), Keys) And Keys.KeyCode
+                Dim KeyCode As Keys = DirectCast(CInt(m.WParam.ToInt64() And Keys.KeyCode), Keys)
```

Comment:
The enum has an underlying type of int so no need to do the AND using the 64-bit value. I'm not sure if the 32-bit value gets extended to 64-bits to do this.

Reply:
I think my assumptions here were wrong. Ignore this comment.

Reply:
This is not that important but it was bugging me enough that I decided to drill into the Release code to see what happens in x64 in a simple Winforms app. If you bitwise AND after casting to an int, only a (32-bit) AND instruction is used:

and         edx,0FFFFh 

In the current pattern, we generate 2 additional instructions - a load and a sign extension to a 64-bit value - before doing a 64-bit AND.

mov         ecx,0FFFFh  
movsxd      rcx,ecx  
and         rcx,qword ptr [rbp-8]  

Not recommending a change here, simply documenting the behavior I observed from what didn't need to become a rabbit hole :)

Reply:
My vote is we fix in Preview 1 as well. Seems like a bad issue to leave in Preview 1.